### ISSUE TITLE
[Bug] CanResize recovers to True automatically after minimizing the window.

### ISSUE THREAD

**Issue by yuzh0816** [kikipoulet/SukiUI#251]
### Check the following items

- [X] I have looked up relevant Issue

### Description of the issue

After setting the CanResize to False, the window seems to act well.
When minimizing the window and recover it, the CanResize status returns to True.

### Package Version

Latest

### Environment

Windows 11 22635

### Expected Behavior

/

### Reproduction

Set CanResize to False

**Comment by AuroraZiling** [comment]
Related code in `SukiWindow.axaml.cs`:
```csharp
private void OnWindowStateChanged(WindowState state)
{
    if (state == WindowState.FullScreen)
        CanResize = CanMove = false;
    else
        CanResize = CanMove = true; // THIS LINE
}
```

**Comment by yuzh0816** [comment]
Fixed in #254
